aerosonicdb/models/mfcc/cnn.py

```diff
@@ -168,54 +192,68 @@ def run_cv(train_path=TRAIN_PATH,
         ap_score = average_precision_score(y_true=y_test, y_score=y_prob)
         env_results.append(ap_score)
 
-        PrecisionRecallDisplay.from_predictions(y_test, y_prob, ax=ax, name=f'CNN {count}')
+        PrecisionRecallDisplay.from_predictions(
+            y_test, y_prob, ax=ax, name=f"CNN {count}"
+        )
 
         count += 1
 
-    ax.legend(loc='upper right')
-    ax.set_title('CNN PR curves: ENV')
+    ax.legend(loc="upper right")
+    ax.set_title("CNN PR curves: ENV")
     ax.grid(linestyle="--")
 
     plt.legend()
-    plt.savefig(f'../figures/CNN_Env_PR_curves.png', dpi=300)
+    plt.savefig(f"../figures/CNN_Env_PR_curves.png", dpi=300)
     plt.show()
 
-    print('Environment evaluation results:', env_results, sep='\n')
+    print("Environment evaluation results:", env_results, sep="\n")
 
     env_mean = np.mean(env_results) * 100
     env_st_dev = np.std(env_results) * 100
-    print(f'Average Precision Score against Environment set: %.2f%% (%.2f%%)' % (env_mean, env_st_dev))
+    print(
+        f"Average Precision Score against Environment set: %.2f%% (%.2f%%)"
+        % (env_mean, env_st_dev)
+    )
     env_scores = (env_mean, env_st_dev, env_results)
 
     return cv_scores, test_scores, env_scores
 
 
-def train_save_model(output_path=OUTPUT_PATH,
-                     train_path=TRAIN_PATH,
-                     filename='mfcc_cnn',
-                     epochs=1,
-                     batch_size=216,
-                     verbose=0,
-                     rand_seed=0):
-    
+def train_save_model(
+    output_path=OUTPUT_PATH,
+    train_path=TRAIN_PATH,
+    filename="mfcc_cnn",
+    epochs=1,
+    batch_size=216,
+    verbose=0,
+    rand_seed=0,
+):
+
     keras.utils.set_random_seed(rand_seed)
 
-    X, y, g = load_train_data(data_path=train_path, target_label='class_label')
+    X, y, g = load_train_data(data_path=train_path, target_label="class_label")
 
     X_train, y_train, X_val, y_val = train_val_split(X, y, g)
 
     model = init_model(X)
     model.summary()
 
     # train model
-    model.fit(X_train, y_train, validation_data=(X_val, y_val), batch_size=batch_size, epochs=epochs, verbose=verbose)
+    model.fit(
+        X_train,
+        y_train,
+        validation_data=(X_val, y_val),
+        batch_size=batch_size,
+        epochs=epochs,
+        verbose=verbose,
+    )
 
     # save the model
-    model_path = os.path.join(output_path, filename, 'model')
+    model_path = os.path.join(output_path, filename, "model.keras")

```

Comment:
without the `.keras` extension this was crashing - keras now requires this extension apparently.